x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/ProfilingIndexTemplateRegistry.java

```diff
@@ -44,10 +44,11 @@ public class ProfilingIndexTemplateRegistry extends IndexTemplateRegistry {
     // version 1: initial
     // version 2: Added 'profiling.host.machine' keyword mapping to profiling-hosts
     // version 3: Add optional component template 'profiling-ilm@custom' to all ILM-managed index templates
-    public static final int INDEX_TEMPLATE_VERSION = 3;
+    // version 4: Added 'service.name' keyword mapping to profiling-events
+    public static final int INDEX_TEMPLATE_VERSION = 4;
 
     // history for individual indices / index templates. Only bump these for breaking changes that require to create a new index
-    public static final int PROFILING_EVENTS_VERSION = 1;
+    public static final int PROFILING_EVENTS_VERSION = 2;
```

Comment:
We bump this despite not doing any breaking changes because we expect a rollover on this index to be cheap and want to make sure that the field becomes available immediately after upgrading a cluster to 8.13.